README.Mac.md

```diff
@@ -1,294 +1,281 @@
-
-```#LoadModule userdir_module libexec/apache2/mod_userdir.so ```
-and
-``` #Include /private/etc/apache2/extra/httpd-userdir.conf ```
+**c.** Add the below lines under </IfModule>:
+```
+<FilesMatch \.php$>
+    SetHandler application/x-httpd-php
+</FilesMatch>
+```
 
-At the end of your file also include the line
-``` Include /etc/apache2/other/*.conf ```
+#### Restart apache and it will reload the updated settings
+```
+sudo apachectl -k restart
+```
 
-Now open ```/etc/apache2/extra/httpd-userdir.conf``` and uncomment the following line: ```#Include /private/etc/apache2/users/*.conf```.
+## 1.7 (Optional) Installing Xdebug
 
-## 3.2 Enable PHP
-Open  ```/etc/apache2/httpd.conf``` in a text editor and locate the line containing
-```LoadModule php5_module libexec/apache2/libphp5.so```
-Ensure that the line is uncommented (remove the ```#``` at the beginning of the line)
+Xdebug allows debugging with breakpoints (such as when using an IDE ex. Intellij).
 
-## 3.3 Setup MySql with Apache2
-Do the following to allow for use of MySql with Apache2, unless link already exists
+Paste into your terminal and hit enter:
 ```
-sudo mkdir /var/mysql
-sudo ln -s /tmp/mysql.sock /var/mysql/mysql.sock
+pecl install xdebug
 ```
 
-## 3.4 Setting up user conf file
+### 1.7.1 Configure Xdebug for local development
 
-In order for apache to know where to find the LORIS code base you have to first set up your user config file. To do so, open your user conf file using the following in the ``` /etc/Apache2/Users/``` directory:
+**a.** Comment out the line `zend_extensions=“xdebug.so”` from the top of your php.ini (Note: replace your_php_version with the correct php version installed):
+```
+sudo open -e /usr/local/etc/php/<your_php_version>/php.ini
+```
 
+b. Create the file ext-xdebug.ini with this command:
 ```
-sudo vi $username.conf
+sudo touch /usr/local/etc/php/<your_php_version>/conf.d/ext-xdebug.ini
 ```
-<i>$username ⇾ your username for the computer</i>
 
-Once in the text editor, ensure that the file looks similar to the following:
+c. Paste the following into the ext-xdebug.ini file you just created (Note: replace the zend_extension path with the correct one on your machine):
+```
+[xdebug]
+zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
+xdebug.remote_enable=on
+xdebug.remote_host=localhost
+xdebug.remote_port=9000
+xdebug.show_error_trace=1
+xdebug.remote_autostart=1
+xdebug.remote_connect_back=0
+xdebug.idekey=phpstorm
+```
 
+#### Restart apache and it will reload with Xdebug configured.
 ```
-<Directory "/Users/$username/Sites/">
-    Options Indexes MultiViews FollowSymLinks
-    AllowOverride None
-    Order allow,deny
-    Allow from all
-</Directory>
+sudo apachectl -k restart
 ```
 
-**NOTE FOR OSX10/Apache 2.4 users:** 
+### 1.7.2 (Optional) Install [Xdebug toggle for OSX](https://github.com/w00fz/xdebug-osx)
```

Comment:
maybe instead of the term Optional at the beginning of the title, I would replace it by (recommended) at the end of the title like it is done for section 1.8.1